integration-tests/metrics/k8s_metric.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import print_function
+
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+import pprint
+
+import numpy as np
+import requests
+import yaml
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+sys.path.insert(0, '{}/../'.format(cur_dir))
+from test_utils import log_clipper_state, create_kubernetes_connection
+from metric_utils import parse_res_and_assert_node, get_matched_query, get_metrics_config
+
+sys.path.insert(0, os.path.abspath("%s/../../clipper_admin" % cur_dir))
+from clipper_admin import ClipperConnection
+from clipper_admin.deployers import python as python_deployer
+
+
+def predict(addr, x):
+    url = "http://%s/simple-example/predict" % addr
+    req_json = json.dumps({'input': list(x)})
+    headers = {'Content-type': 'application/json'}
+    r = requests.post(url, headers=headers, data=req_json)
+    if r.status_code == requests.codes.ok:
+        print(r.json())
+
+
+def feature_sum(xs):
+    return [str(sum(x)) for x in xs]
+
+
+def log_docker_ps(clipper_conn):
+    container_runing = clipper_conn.cm.docker_client.containers.list()
+    logger.info('Current docker status')
+    for cont in container_runing:
+        logger.info('Name {}, Image {}, Status {}, Label {}'.format(
+            cont.name, cont.image, cont.status, cont.labels))
+
+
+if __name__ == '__main__':
+    query_request_template = "http://{}/api/v1/series?match[]={}"
+
+    logging.basicConfig(
+        format=
+        '%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+        datefmt='%y-%m-%d:%H:%M:%S',
+        level=0)
+
+    logger = logging.getLogger(__name__)
+
+    logger.info("Start K8s Metric Test (0/1): Running 2 Replicas")
+    clipper_conn = clipper_conn = create_kubernetes_connection(
```

Comment:
You have a double `clipper_conn = ` here

Reply:
Fixed.